### DISCUSSION TITLE
Terminal data recovery

### DISCUSSION THREAD

**Discussion by ahmedelhanafy2022** [UvaCsl/HemoCell#76] [General discussion]
Hello all,
Is there any way to recover the data which appears in the terminal such as the rel. app. viscosity, average velocity, etc... ?

**Comment by gzavo** [comment]
Hi @ahmedelhanafy2022,
HemoCell generates log files (*.log) with some of the information, depending on the log-level that you define.
Check the `logDirectory` and  `logFile` tags in your `config.xml`.
